### PR TITLE
add log priority option for stdout log

### PR DESCRIPTION
**Pull request by antonilol**
add log priority option to stdout log (`console.log`) just like the one for syslog
by default, no logs are suppressed. users can manually suppress debug log by setting this option to `info`

maintainers, if you want another name than `STDOUT_LOG_MIN_PRIORITY`, feel free to change it

got the idea from #569

### CODE REVIEW

**Comment by wiz** [comment]
Thanks for this PR, will review today

**Review by wiz — APPROVED** [review]
Tested ACK @ 3ceab1493ece2b57d51923ec2339fe57c75f21dc
Works well, however you need to add the Dockerfile configuration variables and also document this feature in the README before we can merge this PR

File: docker/backend/start.sh [hunk 1]
```diff
@@ -17,6 +17,7 @@ __MEMPOOL_INDEXING_BLOCKS_AMOUNT__=${MEMPOOL_INDEXING_BLOCKS_AMOUNT:=1100}
 __MEMPOOL_PRICE_FEED_UPDATE_INTERVAL__=${MEMPOOL_PRICE_FEED_UPDATE_INTERVAL:=3600}
 __MEMPOOL_USE_SECOND_NODE_FOR_MINFEE__=${MEMPOOL_USE_SECOND_NODE_FOR_MINFEE:=false}
 __MEMPOOL_EXTERNAL_ASSETS__=${MEMPOOL_EXTERNAL_ASSETS:=[]}
+__MEMPOOL_STDOUT_LOG_MIN_PRIORITY__=${MEMPOOL_STDOUT_LOG_MIN_PRIORITY:=[]}
```
Comment (knorrium) [inline docker/backend/start.sh:20]:
```suggestion
__MEMPOOL_STDOUT_LOG_MIN_PRIORITY__=${MEMPOOL_STDOUT_LOG_MIN_PRIORITY:=debug}
```

File: docker/backend/mempool-config.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "MEMPOOL_BLOCKS_AMOUNT": __MEMPOOL_MEMPOOL_BLOCKS_AMOUNT__,
     "PRICE_FEED_UPDATE_INTERVAL": __MEMPOOL_PRICE_FEED_UPDATE_INTERVAL__,
     "USE_SECOND_NODE_FOR_MINFEE": __MEMPOOL_USE_SECOND_NODE_FOR_MINFEE__,
-    "EXTERNAL_ASSETS": __MEMPOOL_EXTERNAL_ASSETS__
+    "EXTERNAL_ASSETS": __MEMPOOL_EXTERNAL_ASSETS__,
+    "STDOUT_LOG_MIN_PRIORITY": __MEMPOOL_STDOUT_LOG_MIN_PRIORITY__
```
Comment (knorrium) [inline docker/backend/mempool-config.json:18]:
```suggestion
    "STDOUT_LOG_MIN_PRIORITY": "__MEMPOOL_STDOUT_LOG_MIN_PRIORITY__"
```

**Review by knorrium — DISMISSED** [review]
Please make these two small changes since the log level is a string

**Comment by antonilol** [comment]
oops

**Comment by wiz** [comment]
can you please fix the merge conflicts and we can re-review

**Pull request merged**